python/ray/autoscaler/commands.py

```diff
@@ -268,10 +278,42 @@ def get_or_create_head_node(config, config_file, no_restart, restart_only, yes,
                   quote(monitor_str), modifiers))
         print("To open a console on the cluster:\n\n"
               "  ray attach {}{}\n".format(config_file, modifiers))
+
         print("To ssh manually to the cluster, run:\n\n"
               "  ssh -i {} {}@{}\n".format(config["auth"]["ssh_private_key"],
                                            config["auth"]["ssh_user"],
-                                           provider.external_ip(head_node)))
+                                           head_node_ip))
+
+        def print_dashboard_url():
```

Comment:
I don't think this should be tied with the autoscaler, rather it should be printed on ray init (similar to the "old" web UI url).

Reply:
Makes sense, will look at this tomorrow.

Reply:
Should be fixed now. The non-AS method prints a localhost url; the actual IP is written to a file (in house we're rsyncing this file down to get the url out).